src/puppetlabs/puppetdb/query_eng/engine.clj

```diff
@@ -1473,9 +1556,13 @@
             [["not" expression]]
             (map->NotExpression {:clause (user-node->plan-node query-rec expression)})
 
-            [["in" column subquery-expression]]
-            (map->InExpression {:column (columns->fields query-rec (utils/vector-maybe column))
-                                :subquery (user-node->plan-node query-rec subquery-expression)})
+            [["in" column-s subquery-expr]]
```

Comment:
column or columns, lol ... I'd just call it columns, I can see that hyphen annoying people :-).